inc/Engine/Optimization/LazyRenderContent/Frontend/Processor/Dom.php

```diff
@@ -10,6 +10,24 @@ class Dom implements ProcessorInterface {
 
 	use HelperTrait;
 
+	/**
+	 * Number of injects hashes.
+	 *
+	 * @since 3.6
```

Comment:
It's been added here, so why is it `@since 3.6` ? Shouldn't it be `3.17` ?

Reply:
@Miraeld Good catch thanks, it's a bad copy/paste.
For such feedback, do not hesitate to make the change directly to avoid further delay